### ISSUE TITLE
Cluster stuck in non-ready state because channel update rejected

### ISSUE THREAD

**Issue by jlewi** [kubeflow/gcp-blueprints#35]
While deploying GCP private clusters I encountered a problem where the command

```
kubectl --context=gcp-private-dev-mgmt wait --for=condition=Ready --timeout=600s  containercluster gcp-private-0527
```
would timeout. It turned out the problem was that the last state for the cluster

was

```
  - lastTransitionTime: "2020-05-29T00:15:38Z"
    message: 'Update call failed: the desired mutation for the following field(s)
      is invalid: [releaseChannel.0.Channel]'
    reason: UpdateFailed
    status: "False"
    type: Ready
```

gcloud describe however shows the cluster to be in the stable channel. So not sure why the update would fail.

```
gcloud beta --project=$PROJECT container clusters describe --region=$REGION $CLUSTER 
...
releaseChannel:
  channel: STABLE
```

**Comment by jlewi** [comment]
It doesn't seem like kubectl wait supports waiting for multiple conditions.

**Comment by jlewi** [comment]
The easiest work around for no is just to comment out the `kubectl wait` command in the Makefile so that it will keep going.

**Comment by jlewi** [comment]
I tried a fresh deployment and I got the same error so I think it might be an issue with CNRM.
Filed
GoogleCloudPlatform/k8s-config-connector#194

**Comment by jlewi** [comment]
Should be fixed by /kubeflow/manifests#1230
